tgui/packages/tgui/format.js

```diff
@@ -110,3 +110,10 @@ export const formatDb = value => {
   }
   return sign + formatted + ' dB';
 };
+
+/**
+ * Truncates a string with an ellipsis after n characters. Default is 25.
+ */
+export const truncate = (str, n = 25) => {
```

Comment:
What's the use-case for this over `text-overflow: ellipsis`? https://developer.mozilla.org/en-US/docs/Web/CSS/text-overflow

Reply:
that not working

Reply:
That worries me and should be looked at. Can you create an Issue for that? On phone atm.

Reply:
valid function for when you don't want to do it for overflowing imo

Reply:
Disagree due to variable character widths, but we can argue about that another time.